specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -155,17 +219,27 @@ SECRET_VALUE.
   @doc("The load test configuration.")
   loadTestConfiguration?: LoadTestConfiguration;
 
+  @added(APIVersions.v2023_04_01_preview)
+  @doc("Id of the test run to be used as the baseline reference for results trends.")
+  baselineTestRunId?: string;
+
   @doc("The input artifacts for the test.")
   @visibility("read")
   inputArtifacts?: TestInputArtifacts;
 
   @doc("Unique test name as identifier.")
+  @maxLength(50)
```

Comment:
nit: should the model properties be ordered by priority for readabililty. elg. testid, displayName, description, and then other properties from mandatory to optional